source/CardGames.py

```diff
@@ -155,4 +176,28 @@ def dealCards(self, numCards: int, players: "list[Player]"):
 
   def resetDeck(self):
     self.deck.reset()
-    self.deck.shuffle()
\ No newline at end of file
+    self.deck.shuffle()
+
+
+# has_pair function---------------------------------------------------------------
+
+def has_pair(player):
+    Hand = player.hand
+    values = [card.value for card in Hand]
+    compare = set()
+    print(values, compare)
+    for i in values:
+        if i in compare:
+            return True
+        compare.add(i)
+    else:
+        return False
+#---------------------------------------------------------------------------------
+
+
+# call function--------------------------------------------------------------
+
+def Call(player, bet: int, pot):
+  player.makeBet(bet, pot)
+
+#---------------------------------------------------------------------------------
```

Comment:
Make sure to properly indent from lines 184 to 200 so that they are within the Dealer class, for better readability! 

Overall, I like how efficiently the functions are working, starting from the makeBet function being able to update the pot class, and finally, the Call function invoking the makeBet function :)

Reply:
Thank you for the input!